metrics/zio-telemetry-open-tracing-backend/src/main/scala/sttp/client/ziotelemetry/opentracing/ZioTelemetryOpenTracingBackend.scala

```diff
@@ -0,0 +1,49 @@
+package sttp.client.ziotelemetry.opentracing
+
+import io.opentracing.propagation.{Format, TextMapAdapter}
+
+import scala.jdk.CollectionConverters._
+import sttp.client._
+import sttp.client.impl.zio.RIOMonadAsyncError
+import sttp.client.monad.MonadError
+import sttp.client.ws.WebSocketResponse
+import zio._
+import zio.telemetry.opentracing._
+
+class ZioTelemetryOpenTracingBackend[-WS_HANLDER[_]](other: SttpBackend[Task, Nothing, WS_HANLDER], requestTagCollector: RequestTagCollector = RequestTagCollector.empty) extends SttpBackend[RIO[OpenTracing, *], Nothing, WS_HANLDER] {
+
+  @SuppressWarnings(Array("scalafix:Disable.toString"))
+  def send[T](request: Request[T, Nothing]): RIO[OpenTracing, Response[T]] = {
+    val headers = scala.collection.mutable.Map.empty[String, String]
```

Comment:
so the mutable `headers` map is to pass the injected opentracing headers to the request?

Reply:
Yes it creates from the existing Span context a set of headers which are put on the request such that distributed tracing is enabled